HISTORY.rst

```diff
@@ -1,6 +1,28 @@
 Release History
 ---------------
 
+0.8.0 (2019-10-??)
+++++++++++++++++++
+Major release with several breaking changes.
+
+* API change: Dropped support for Python 2
+* API change: Gathering of git information gathering is now enabled by default #595
+* API change: Switched constructor from Observer.create(...) to Observer(...) for all observers.
+* API change: Changed the interface for collecting custom host-information #569
+* API change: Changed interface for defining CLI options. #572
+* API change: Started using pathlib.Path instead of os.path in many places
+* Feature: Added new S3 file observer #542
+* Feature: added `started_text` option to TelegramObserver #494
+* Feature: added copy/deepcopy support to read-only containers #500
+* Bugfix: FileStorage Observer is more reliable under parallel execution #503
+* Bugfix: FileStorageObserver now raises an error when an artifact would overwrite an important file #647
+* Bugfix: fixed inconsistent config nesting behavior #409 #505
+* Bugfix: Several fixes for tensorflow integration
+* Bugfix: Fixed crash due to missing brand-key on some machines # 512
+* Internal: Migrated CI server to Azure
+* Internal: Added pre-commit hooks for python black for automated code formatting
+
```

Comment:
Do you usually put doc improvement in the release ?
(There some minor typo correction) (Is there doc/example for the new feature and observer ?)

Also, I was reading the PR and I noticed that there two PR mistake as of the create method for the observer (kairos03 and me). It seem to have an inconsistency between the doc and the release code. In this release is this problem will be fix ?

Reply:
I don't usually mention doc changes in the release notes. 
They are meant for people who know sacred already and just want to stay up to date on what has changed.

Reply:
@Qwlouse and fot the doc of the observer ?